balancer-design.typ

```diff
@@ -17,3 +17,46 @@ In Rust, packages are called "crates". The Balancer and Harness is split into mu
 	image("figures/balancer-crates.svg"),
 	caption: "Package diagram showing the Balancer and Harness crates.",
 ) <Figure::balancer-crates>
+
+== Dependencies
+
+#let dependencies = toml("Cargo.toml")
+
+#figure(
+	table(
+		[*Dependencies*],
+		[#dependencies.workspace]
+	),
+	caption: "External dependencies required by the balancer"
+)
```

Comment:
This table is formatted really poorly, and Darian is gonna complain about it. It also includes other metadata from the top level of the cargo.toml, which is irrelevant. 

What I'm looking for is a table that looks something like this:

| Crate   | Version | Features  |
|---------|---------|-----------|
| anyhow  | 1       |           |
| figment | 0.10.10 | toml, env |
| ...     |         |           |

Reply:
I'll figure out formatting after break